src/app/packages/menu-items/tools/services/go-to.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import * as proj4 from 'proj4';
+import { isEqual as _isEqual } from 'lodash';
+
+@Injectable()
+export class GoToService {
```

Comment:
Move to external function that can be consumed in other places (core/utils for example)